.nix/fallback-config.nix

```diff
@@ -24,6 +24,7 @@ with (import (import ./nixpkgs.nix) {}).lib;
     })) // {
     master = {
       coqPackages.coq.override.version = "master";
+      coqPackages.coq.override.native = true;
```

Comment:
You've overridden the "native" argument for the build of Coq master only, which does not include any Coq packages, so this is not surprising that we are not seeing any failure.

Reply:
How can I override all the pacakges?